utils_test.go

```diff
@@ -0,0 +1,81 @@
+package hdfs
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/beyondstorage/go-endpoint"
+	ps "github.com/beyondstorage/go-storage/v4/pairs"
+	"github.com/beyondstorage/go-storage/v4/services"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func NewClient(t *testing.T) *Storage {
```

Comment:
How about move this function into `tests` and rename to `setup` like https://github.com/beyondstorage/go-service-s3/blob/master/tests/utils_test.go#L14 ?